assignment4/app.js

```diff
@@ -0,0 +1,26 @@
+const fs = require("fs");
+
+var fileArray = ["file1.txt", "file2.txt", "file3.txt"];
+
+function printData(err, data) {
+  if (err) console.log(err);
+  else console.log(data);
+}
+fs.readFile("file1.txt", "utf-8", function(err, data) {
+  printData(err, data);
+
+  fs.appendFile("file2.txt", data, function(err) {
+    if (err) console.log(err);
+    else console.log("Appended File 1 to file 2");
```

Comment:
Instead of having if-else block here, use if-return strategy. i.e. instead of
```javascript
if (condition)
  statement 1
else
  statement 2
```

Write like:
```javascript
if (condition)
  return statement 1;
statement 2;
```